sandbox/rocky/tf/misc/tensor_utils.py

```diff
@@ -118,3 +120,25 @@ def pad_tensor_dict(tensor_dict, max_len):
         else:
             ret[k] = pad_tensor(tensor_dict[k], max_len)
     return ret
+
+
+class enclosing_scope(object):
+    def __init__(self, primitive_name, sys_op_name, **kwargs):
```

Comment:
primitive_name --> enclosing_name
sys_op_name --> name

...and similarly for the internal variables.